jmxtrans-core/pom.xml

```diff
@@ -179,5 +172,23 @@
 			</plugin>
 		</plugins>
 	</build>
-
+	<profiles>
+		<profile>
+			<id>toolsJar</id>
+			<activation>
+				<file>
+					<exists>${toolsjar}</exists>
+				</file>
+			</activation>
+			<dependencies>
+				<dependency>
+					<groupId>com.sun</groupId>
```

Comment:
I'm not sure where this dependency came from and if we even need it anymore. Could we just drop it? Or could you add a comment on why it is needed (if you know)?

Reply:
We can't drop it see issue #649 where I not the problems I encounter.

Reply:
Of course!